apps/financial-aid/backend/src/app/modules/staff/staff.service.ts

```diff
@@ -33,6 +34,9 @@ export class StaffService {
       where: {
         municipalityId,
       },
+      order: Sequelize.literal(
+        'CASE WHEN active = true THEN 0 ELSE 1 END, name ASC',
```

Comment:
You can also do   order: [['active', 'DESC']] I think 💭

Reply:
Didn't know that, but do you think it is more readable?